.github/workflows/ci-rust.yml

```diff
@@ -0,0 +1,35 @@
+name: 'CI - Rust'
+
+on:
+  pull_request:
+    branches:
+      - trunk
```

Comment:
I think we can leave the branch out, as we'd like to run tests when code is pushed in a "work-in-progress" branch.

Reply:
I have just committed a second patch reming these lines.